server/shared/types/dataset.ts

```diff
@@ -308,7 +308,9 @@ export type MetaboliteIntensityQueryNative = {
 	src: 'native' | string
 	file: string
 	samples?: number[]
+	_metabolites?: string[]
```

Comment:
in type def, always add comment on every property. here indicate this is a dynamically built cache of string names to speed up search